integration/cri-o/cri-o.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/crio_skip_tests.sh"
+
+crio_repository="github.com/kubernetes-incubator/cri-o"
+runtime_repository="github.com/clearcontainers/runtime"
+
+echo "Obtain CRI-O supported version"
+go get -u -d "${runtime_repository}" || true
+versions_file="$GOPATH/src/${runtime_repository}/versions.txt"
```

Comment:
sorry, forgot to mention that we already have a function for getting the versions:
```
source .ci/lib.sh
get_cc_versions
```